crates/networking/p2p/net.rs

```diff
@@ -80,6 +92,7 @@ async fn discover_peers(
         storage,
         table.clone(),
         signer.clone(),
+        connection_broadcast,
```

Comment:
Maybe it's time to add a `RLPxConnContext` struct to put together the table, signer, storage and channel(s) in a single parameter.

(If not here, maybe open a ticket to deal with it)

Reply:
Opened an issue [here](https://github.com/lambdaclass/lambda_ethereum_rust/issues/1140) for it.


Maybe I'll add it naturally on future P2P PRs if I see it's becoming hard to handle with arguments alone.